nicenumber/nicenumber.py

```diff
@@ -166,6 +166,36 @@ def to_color(number:int, color:list = None):
     -------
     str
         String number with different colors
+
```

Comment:
Could allow user to pass in custom palette of `dict(color1=111, color2=222)`. Not super necessary but could be a nice feature.